### ISSUE TITLE
To narrow confidence and credible intervals for main effects in repeated measures anova

### ISSUE THREAD

**Issue by AugustOlsson** [jasp-stats/jasp-desktop#2368]
OS name and version: Windows 10 Education
Analysis: Repeated measures ANOVA/ Baysian repreated measures ANOVA with between subject factor

Hi,

I suspect that Jasp is producing to narrow confidence and credible intervals for main effects of a between variable in repeated measures ANOVA and BANOVA. I have a design with 35 subjects (16 in one between subject condition 19 in the other). These subjects have each made 28 different judgements within-participants. A repeated measures ANOVA shows that the main effect of condition (between subject variable) is non-significant, however when I produce a graph over the main effect the confidence intervals are not overlapping, showing a clear significant effect. When I compute the same analysis in another software (STATISTICA) I get the same F and P-value, but the confidence intervals are clearly overlapping. This is expected with non-significant results. Similarily the credible intervals for the main effect of condition when computing a Baysian repeated measure anova look similar to the confidence intervals produced in Jasp, which makes me suspect that they are to narrow as well. 

I couldn't find a post on any bug regarding confidence and credible intervals. Is this a known issue? Are there any quick fixes?

Best,
August

Edit, here are the stats and the graph over the mean of each between condition with 95% confidence interval in JASP and STATISTICA. I have edited the STATISTICA graph so that the Y-axis is the same. The conditions are "Kvant" and "Kval" and they are placed on different sides (sorry for this). The mean values are the same, the only difference are the confidence intervals.
![image](https://user-images.githubusercontent.com/36534473/36539984-ec679f00-17d8-11e8-89bb-21c60c700fa0.png)
![image](https://user-images.githubusercontent.com/36534473/36539994-f5047c8c-17d8-11e8-95cf-0ae644c2c90d.png)

**Comment by JohnnyDoorn** [comment]
Hi August,
Thank you for taking the time to report this issue. Could you please provide a .jasp file with the results you mentioned, or the data-file itself? You can either upload it here (as a zipfile), or email it to me at j.b.vandoorn<at>uva.nl
This way I can try to reproduce the issue, and take a look at what is happening under the hood to cause this discrepancy in the plotting.
Kind regards,
Johnny

**Comment by AugustOlsson** [comment]
What is your email adress Johnny? Did you mean to write j.b (at) vandoornuva.nl?

**Comment by JohnnyDoorn** [comment]
Sorry, the markup must have messed it up a little bit - it's j.b.vandoorn (at) uva.nl

**Comment by JohnnyDoorn** [comment]
Hi August,
I have looked at the code, and it appears to be an issue of whether to pool the standard deviation across the RM factors. In the current setup, the sd is taken over all levels of the RM factors. It might be the case that Statistica first averages across the levels of the RM factors, and then calculates the sd for each between factor. I have tweaked the JASP code so that it does the latter - could you have a look at the plot it now produces for the dataset you sent me and see if it is more in line with Statistica? 
If so then we have to discuss with the team what would be the best option - the literature seems to be a bit dispersed on the topic of pooling standard deviations in such mixed ANOVA setups.

![rmplotci](https://user-images.githubusercontent.com/15704203/36555782-50ff5992-1803-11e8-9907-18881c3aea08.png)

Kind regards,
Johnny

**Comment by AugustOlsson** [comment]
Hi Johnny,

Thank you very much for the quick response. After consulting a senior collegue I suspected that the deviation could be related to how the SD was computed. The Statistica graph looks identical to me, see below. The question of which is the correct way to do it is as you say not obvious. An old by still cited view of the matter within my field (psychology) is: https://link.springer.com/article/10.3758/BF03210951  

PS. As I wrote in my original post I suspect that the same difference exists for credible intervals when conducting bayesian repeated measures ANOVAS. That is, that credible intervals are calculated based on variance over all RM factors rather than over an average across the RM factors and then between. So, the discussing on how the confidence intervals should be created in these situations should also include how credible intervals should be created under the same circumstances.

![image](https://user-images.githubusercontent.com/36534473/36586110-bbc5a59c-1880-11e8-805b-deb42b169b2a.png)
